recipes/kafka.rb

```diff
@@ -34,7 +34,7 @@
   elsif node['cdap']['cdap_site'].key?('kafka.log.dir')
     node['cdap']['cdap_site']['kafka.log.dir']
   else
-    '/tmp/kafka-logs'
+    '/data/cdap/kafka-logs'
   end
 
 node.default['cdap']['cdap_site']['kafka.server.log.dirs'] = kafka_log_dirs
```

Comment:
note this line would not be a change, it would be a fix in case the user does the unexpected... it forces it to a string.